hawkbit-dmf-amqp/src/test/java/org/eclipse/hawkbit/integration/listener/ReplyToListener.java

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright (c) 2015 Bosch Software Innovations GmbH and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.hawkbit.integration.listener;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.eclipse.hawkbit.AmqpTestConfiguration;
+import org.eclipse.hawkbit.dmf.amqp.api.EventTopic;
+import org.eclipse.hawkbit.dmf.amqp.api.MessageHeaderKey;
+import org.springframework.amqp.core.Message;
+import org.springframework.amqp.rabbit.annotation.RabbitListener;
+
+public class ReplyToListener extends AbstractTestRabbitListener {
+
+    public static final String LISTENER_ID = "replyto";
+
+    private final Map<EventTopic, Message> messages = new HashMap<>();
```

Comment:
This looks somehow strange: Due to the inheritance this class has `getMessage()` and `getMessages()` but they return different content!